### PR TITLE
Use IPv6 by default.

### PR DESCRIPTION
**Pull request by russellb**
Set USE_IPV4=true in config to switch back to IPv4.

### CODE REVIEW

**Comment by metal3ci** [comment]
Build FAILURE, see build http://10.8.144.11:8080/job/dev-tools/1609/

**Comment by russellb** [comment]
> Build FAILURE, see build http://10.8.144.11:8080/job/dev-tools/1609/

this almost passed CI and only failed at the very last thing run_ci.sh does.

```
+(./run_ci.sh:154): wait_for_worker(): '[' 'worker-0.ostest.test.metalkube.org   Ready    worker   13m   v1.17.1' = '' ']'
+(./run_ci.sh:155): wait_for_worker(): TIMEOUT_MINUTES=15
+(./run_ci.sh:156): wait_for_worker(): echo 'worker-0 registered, waiting 15 minutes for Ready condition ...'
+(./run_ci.sh:157): wait_for_worker(): oc wait node/worker-0 --for=condition=Ready --timeout=900s
Error from server (NotFound): nodes "worker-0" not found
```

It was expecting a Node named worker-0 but it's worker-0.ostest.test.metalkube.org  -- not sure when that behavior changed exactly

File: common.sh [hunk 1]
```diff
@@ -58,7 +58,7 @@ export SSH_PUB_KEY="${SSH_PUB_KEY:-$(cat $HOME/.ssh/id_rsa.pub)}"
 # mirror images for installation in restricted network
 export MIRROR_IMAGES=${MIRROR_IMAGES:-}
 
-export USE_IPV4=${USE_IPV4:-true}
+export USE_IPV4=${USE_IPV4:-false}
```
Comment (stbenjam) [inline common.sh]: Do we want to take this opportunity now to make this more versatile? e.g. IP_STACK=ipv6/ipv4/dualstack?

It'd avoid a change in the interface later when we have to do dual stack

**Comment by metal3ci** [comment]
Build SUCCESS, see build http://10.8.144.11:8080/job/dev-tools/1611/

File: common.sh (hunk 1 above)
Comment (russellb) [reply common.sh]: OK, sure, not a big deal

File: 02_configure_host.sh [hunk 2]
```diff
@@ -33,7 +33,7 @@ if [ ! -z "${VM_NODES_FILE}" ]; then
   exit 1
 fi
 
-if [[ "${USE_IPV4}" != "true" ]]; then
+if [[ "${IP_STACK}" != "v6" ]]; then
```
Comment (stbenjam) [inline 02_configure_host.sh]: Should be `==` or `!= "v4"` no?

Comment (russellb) [reply 02_configure_host.sh]: yes, good catch.  thank you.  i pushed before testing

**Comment by stbenjam** [comment]
My local test looks good :shipit:

**Comment by metal3ci** [comment]
Build SUCCESS, see build http://10.8.144.11:8080/job/dev-tools/1615/

**Pull request merged**